### DISCUSSION TITLE
Change img size

### DISCUSSION THREAD

**Discussion by naseemkhan7021** [community/community#24176] [New to GitHub]
<p>Please help, I am trying to resize my image in GitHub ReadMe.md<br>
but i am anable to do this.<br>
Please help me to resize my images in ReadMe.md.</p>

**Comment by Jme7** [comment]
<p>Could you give us the link to your repository? (e.g. <a href="https://github.com/github/gh-ost" rel="nofollow noopener">https://github.com/github/gh-ost</a>)</p>

**Comment by SethClydesdale** [comment, marked as answer]
<p>Hello <a class="mention" href="/naseemkhan7021">@naseemkhan7021</a>,</p>
<p>As far as I know there’s only two methods of resizing images for your readme:</p>
<ol>
<li>the HTML <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img" rel="nofollow noopener"><code>&lt;img&gt;</code></a> tag.</li>
<li>editing the size of the image with an image editing program.</li>
</ol>
<p>The simplest solution would be to use the HTML image tag and add the width/height attributes.</p>
<pre><code class="lang-auto">&lt;img src="IMG_LINK" width="100" height="100"/&gt;
</code></pre>
<p>Replace <code>IMG_LINK</code> with the link of your image, and adjust the width/height values as needed.</p>

**Comment by Jme7** [comment]
<aside class="quote no-group" data-username="SethClydesdale" data-post="3" data-topic="125949">
<div class="title">
<div class="quote-controls"></div>
 SethClydesdale:</div>
<blockquote>
<p>As far as I know there’s only two methods of resizing images for your readme:</p>
<ol>
<li>the HTML <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img" rel="nofollow noopener"> <code>&lt;img&gt;</code> </a> tag.</li>
<li>editing the size of the image with an image editing program.</li>
</ol>
<p>The simplest solution would be to use to HTML image tag and add the width/height attributes.</p>
<pre><code class="lang-auto">&lt;img src="IMG_LINK" width="100" height="100"/&gt;
</code></pre>
</blockquote>
</aside>
<p>[facepalm] I was thinking of that but I forgot that markdown supports html! Thanks <a class="mention" href="/SethClydesdale">@SethClydesdale</a>!</p>
